data/settings.js

```diff
@@ -966,25 +1057,25 @@ Maximum period after which tracked values are purged from the duplicate
 tracking database.`
 	},
 
-	sieve_editheader_forbid_add: {
+	sieve_editheader_header_forbid_add: {
 		tags: [ 'sieve', 'sieve-editheader' ],
 		plugin: 'sieve',
 		values: setting_types.STRING,
 		seealso: [ '[[link,sieve_editheader]]' ],
-		text: `
+		text: `FIXME
 A space-separated list of headers that cannot be added to the message header.
```

Comment:
various FIXMEs here and elsewhere. In here I'm not sure why it's even a FIXME?

Reply:
copy paste mostly I think.
fixed